src/report.ts

```diff
@@ -77,7 +77,7 @@ export class ReportModule {
             error.response.status === 400 &&
             error.responseBody.indexOf(ERROR_RESTRICTED_CODE) !== -1
         ) {
-            return Promise.reject(new Error(ERROR_RESTRICTED_MESSAGE));
+            error.message = ERROR_RESTRICTED_MESSAGE;
```

Comment:
I'm not really keen on modifying param `error` in this case.
I think current way is ok, or we have to clone to new `error` object.

Reply:
update :)